DirectProgramming/Fortran/EdgeDetection/simple-binary-images/README.md

```diff
@@ -0,0 +1,121 @@
+| What you will learn | How to offload Fortran loops to a GPU
+| Time to complete    | 15 minutes
+
+## Purpose
+This sample demonstrates two Fortran implementations of edge detection:
+
+ 1. img_seg_omp_target.F90 implements edge detection on binary images using ordinary for-loops and OpenMP target directives
+ 2. img_seg_do_concurrent.F90 implements edge detection on binary images using only a DO CONCURRENT loop
+
+The implementations are functionally equivalent. In both cases, the OpenMP runtime library is used to parallelize the
+edge detection loops, regardless of whether they are run on the CPU or offloaded to a GPU.
+
+## Key Implementation Details
+[Using Fortran DO CONCURRENT for Accelerator Offload](https://www.intel.com/content/www/us/en/developer/articles/technical/using-fortran-do-current-for-accelerator-offload.html) provides more detailed descriptions of each example code, and discusses the relative merits of each approach.
+
+## License
+Code samples are licensed under the MIT license. See [License.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/License.txt) for details.
+
+Third party program Licenses can be found here: [third-party-programs.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/third-party-programs.txt)
+
+## Using Visual Studio Code* (Optional)
+
+You can use Visual Studio Code (VS Code) extensions to set your environment, create launch configurations,
+and browse and download samples.
+
+The basic steps to build and run a sample using VS Code include:
+ - Download a sample using the extension **Code Sample Browser for Intel oneAPI Toolkits**.
+ - Configure the oneAPI environment with the extension **Environment Configurator for Intel oneAPI Toolkits**.
+ - Open a Terminal in VS Code (**Terminal>New Terminal**).
+ - Run the sample in the VS Code terminal using the instructions below.
+ - (Linux only) Debug your GPU application with GDB for Intel® oneAPI toolkits using the **Generate Launch Configurations** extension.
+
+To learn more about the extensions, see
+[Using Visual Studio Code with Intel® oneAPI Toolkits](https://www.intel.com/content/www/us/en/develop/documentation/using-vs-code-with-intel-oneapi/top.html).
+
+After learning how to use the extensions for Intel oneAPI Toolkits, return to this readme for instructions on how to build and run a sample.
+
+## Building and Running this sample
+
+> **Note**: If you have not already done so, set up your CLI
+> environment by sourcing  the `setvars` script located in
+> the root of your oneAPI installation.
+>
+> Linux Sudo: . /opt/intel/oneapi/setvars.sh
+>
+> Linux User: . ~/intel/oneapi/setvars.sh
+>
+>For more information on environment variables, see Use the setvars Script for [Linux or macOS](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-development-environment-setup/use-the-setvars-script-with-linux-or-macos.html).
+
+### Running Samples on the DevCloud
```

Comment:
@hagabb Can we remove this section (the one that references DevCloud)? We'll have the correct verbiage for IDC at a later date.

Reply:
I removed the section about the IDC.